test/helpers/config.spec.ts

```diff
@@ -1,10 +1,13 @@
-import {
-  generateCustomHeaders,
-  NextConfig
-} from "../../packages/runtime/src/helpers/config"
 import type { NetlifyPluginOptions } from '@netlify/build'
 
-const netlifyConfig = { build: { command: 'npm run build' }, functions: {}, redirects: [], headers: [] } as NetlifyPluginOptions["netlifyConfig"]
+import { generateCustomHeaders, NextConfig } from '../../packages/runtime/src/helpers/config'
+
+const netlifyConfig = {
+  build: { command: 'npm run build' },
+  functions: {},
+  redirects: [],
+  headers: [],
+} as NetlifyPluginOptions['netlifyConfig']
```

Comment:
Maybe for another PR, but prefer not to cast. Add anything missing if required.

```suggestion
const netlifyConfig: NetlifyPluginOptions['netlifyConfig'] = {
  build: { command: 'npm run build' },
  functions: {},
  redirects: [],
  headers: [],
}
```

Reply:
It seems that for the sake of the test, those key/values are enough, a lot of stuff is missing and it just unblocked things